src/xc.c

```diff
@@ -1046,15 +1046,6 @@ op_ufn : {
     case 0377:
     case377:
       CLARITHEQUAL;
-#ifdef OPDISP
-#ifdef ISC
-    case 0400:
-      goto setup_table; /* to defeat optimizer, so optable exists */
-#elif (DOS && OPDISP)
-    case 0400: goto setup_table;
-#endif /* ISC */
-
-#endif /* OPDISP */
 
 #ifdef I386
     /* to defeat the damn optimizer, make it look like */
```

Comment:
Likewise this section below, which is trying to make references to the error labels...

Reply:
That section might actually be needed if the corresponding code came back into play! But it won't ... but I planned to remove it when I remove the other related code.